src/unix/internal.h

```diff
@@ -106,6 +106,12 @@ int uv__pthread_sigmask(int how, const sigset_t* set, sigset_t* oset);
 # define UV__POLLRDHUP 0x2000
 #endif
 
+#ifdef POLLPRI
+# define UV__POLLPRI POLLPRI
+#else
+# define UV__POLLPRI 0x0002
```

Comment:
I'd set it to zero if the platform doesn't have POLLPRI because then you can do the following in uv_poll_start():
```c
if (UV__POLLPRI == 0 && (events & UV_PRIORITIZED) != 0)
  return UV_EINVAL;
```
POLLPRI changes the semantics of polling for events so it's IMO better to return an error early than to silently ignore it or do the wrong thing.